x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/components/agent_policy_form.tsx

```diff
@@ -170,7 +171,20 @@ export const AgentPolicyForm: React.FunctionComponent<Props> = ({
         description={
           <FormattedMessage
             id="xpack.fleet.agentPolicyForm.namespaceFieldDescription"
-            defaultMessage="Apply a default namespace to integrations that use this policy. Integrations can specify their own namespaces."
+            defaultMessage="Namespaces are part of the data stream naming scheme and make it easier to search for data and manage user permissions. The policy's default namespace is applied to the naming of an integration's data streams. For more information, see the {fleetUserGuide}."
```

Comment:
> The policy's default namespace is applied to the naming of an integration's data streams. For more information, see the {fleetUserGuide}.

This only happens if a user doesn't change the default namespace, right? If they do, it's the policy's namespace that will be applied to naming. What about something like this instead?

```suggestion
            defaultMessage="Namespaces are a user-configurable arbitrary grouping that makes it easier to search for data and manage user permissions. A policy's namespace is used to name an integration's data streams. For more information, see the {fleetUserGuide}."
```

Reply:
Updated based on feedback